Packs/ApiModules/Scripts/TAXII2ApiModule/TAXII2ApiModule.py

```diff
@@ -30,8 +31,14 @@ def filter(self, record):
         return False
 
 
+# Make sure we have only one XsoarSuppressWarningFilter
 v21_logger = logging.getLogger("taxii2client.v21")
-v21_logger.addFilter(SuppressWarningFilter())
+demisto.debug(f'Logging Filters before cleaning: {v21_logger.filters=}')
+for current_filter in list(v21_logger.filters):    # pragma: no cover
+    if 'XsoarSuppressWarningFilter' in type(current_filter).__name__:
+        v21_logger.removeFilter(current_filter)
+v21_logger.addFilter(XsoarSuppressWarningFilter())
+demisto.debug(f'Logging Filters: {v21_logger.filters=}')
```

Comment:
Do you have to remove the filter if you add it later?
Maybe an if condition/function in that case?

Reply:
The root cause is that there are "old" filters, from previous runs, with no real handle to them.
I guess it's the way the `looper` is implemented.